test/jdk/java/util/TimeZone/tools/share/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (c) 2008, 2022, Oracle and/or its affiliates. All rights reserved.
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+# or visit www.oracle.com if you need additional information or have any
+# questions.
+#
+
+#
+#
+# This makefile is used to update the time zone tests data: aliases.txt displaynames.txt
+# and VERSION. After switching to new build system in JDK8 the following make
+# command should be used for test data updates:
+#     make JDK_HOME=<path to built JDK with latest tzdata>
+#     make install
+#
+
+TZDATA = ../../../../../../../../open/src/java.base/share/data/tzdata
```

Comment:
suggest you replace this with `../../../../../../../src/java.base/share/data/tzdata/`